marklogic-data-hub/src/test/java/com/marklogic/hub/deploy/commands/LoadUserArtifactsCommandTest.java

```diff
@@ -205,4 +207,26 @@ public void validateArtifactNames() throws Exception {
             () -> loadUserArtifactsCommand.execute(commandContext));
         assertEquals(errorMessage, ex.getServerMessage());
     }
+
+    @Test
+    void entityModelShouldNotBeLoadedFromSubFolder(){
```

Comment:
I actually don't think we should include a test for this - the behavior of loading from a subfolder is not expected and is inconsistent with all other artifact types. I think deleting the code is sufficient here, we don't need to verify that DHF isn't doing something we don't expect it to do (we'd have an infinite number of tests to write otherwise).

Reply:
I agree, I have removed the test.